superset/assets/src/dashboard/components/IconButton.jsx

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const propTypes = {
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
   className: PropTypes.string,
   label: PropTypes.string,
 };
 
 const defaultProps = {
   className: null,
   label: null,
+  onClick: () => true,
```

Comment:
hmm this is a little strange, but I guess is okay. Is this preferred so you don't have to set styles on a simple `<div className="fa fa-trash" />`?

Reply:
i want to re-use `IconButton` component, and it required a onClick callback. in my case, i want to use IconButton as trigger to show modal, and ModalTrigger will bind onClick itself. I can't pass in onClick callback for this instance.

so here i changed onClick prop not required, and add a default callback. Not sure this is right way.